skelebot/systems/scaffolding/scaffolder.py

```diff
@@ -20,7 +22,22 @@ def __format_variables(self, text):
                 text = text.replace(key_str, value)
 
         return text
-    
+
+    def __format_config(self, config):
+        return eval(self.__format_variables(str(config)))
+
+    def __load_github_template(self, owner, repo, branch="main"):
+        url = GITHUB_RAW.format(owner=owner, repo=repo, branch=branch, filepath="template.yaml")
+        response = requests.get(url)
```

Comment:
Pylint was throwing a warning here about missing the timeout argument: https://pylint.pycqa.org/en/latest/user_guide/messages/warning/missing-timeout.html

Reply:
I can add that, sounds like a good thing to have